docs/usage/self-hosted-experimental.md

```diff
@@ -1,22 +1,23 @@
-# Self-Hosted Experimental Options
+# Self-hosted experimental options
 
-The following options are considered "experimental" in that:
+The following options are "experimental" because:
 
-- They should not commonly be needed
-- They are typically an effort to work around some other service or platform's problem
-- We hope they can each be removed one day, or we are waiting to be sure they work as intended
+- They are not commonly needed
+- They are typically an effort to work around some other service's or platform's problem
+- They can be removed at any time
+- They are options for Renovate's internal use to validate they work as intended
 
-Experimental options which are commonly used and for which there is no external solution in sight should be replaced with official options.
+Experimental options which are commonly used and for which there is no external solution in sight should be replaced with official options ("Who is doing the replacing here? Renovate bot team or end-user?").
```

Comment:
Probably should clarify here who is doing the replacing here, the Renovate team or end-user.

Reply:
By us or a contributor

Reply:
So in essence, if the external option turns out to be a good workaround for a external problem, it can become an official option?

Reply:
yes, it but it should then become an official config option like any other option. Not a simple env variable.

Reply:
```suggestion
Experimental options which are commonly used and for which there is no external solution in sight can be converted to a official configuration option by the Renovate bot developers.
```

How about something like this then?

Reply:
sounds good to me